File: 프로그래머스/src/프로그래머스_02262022/이상한문자_만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s)
{
    string answer = "";
    int cnt = 0;
    for (int i = 0; i < s.size(); i++)
    {
        if (s.at(i) == ' ')
            cnt = 1;
        if (cnt % 2 == 0)
            answer += toupper(s.at(i));
        else
            answer += tolower(s.at(i));
        cnt++;
    }
    return answer;
}

File: 2026/Level1/이상한문자만들기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/12930?language=python3
def solution(s):
    answer = ""
    cnt = 0
    
    for x in s :
        if x == ' ':
            answer += x
            cnt = 0           # 단어 바뀌면 초기화
            continue
        
        if cnt %2 == 0 :
            answer += x.upper()
        else :
            answer += x.lower()
        cnt += 1
    return answer